src/myspace/MyPortfolio.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function MyPortfolio() {
```

Comment:
Again I just find My My in everything unnecessary.

Reply:
done. now without `My`